packages/data-point/lib/reducer-types/resolve.js

```diff
@@ -10,6 +11,7 @@ const { DEFAULT_VALUE } = require('./reducer-symbols')
 
 const reducers = Object.assign({}, ReducerHelpers, {
   [ReducerEntity.type]: ReducerEntity,
+  [ReducerEntityInstance.type]: ReducerEntityInstance,
```

Comment:
can you explain why there's a `ReducerEntity` and a `ReducerEntityInstace`? is that for backwards compatibility?